src/components/CheckEditor/CheckEditor.tsx

```diff
@@ -71,29 +84,25 @@ export const CheckEditor: FC<Props> = ({ check, instance, onReturn }) => {
     <FormContext {...formMethods}>
       <form onSubmit={formMethods.handleSubmit(onSubmit)}>
         <Legend>{check?.id ? 'Edit Check' : 'Add Check'}</Legend>
-        <div
-          className={css`
-            margin-bottom: 8px;
-          `}
-        >
+        <div className={styles.marginBotton}>
```

Comment:
Just an idea - With something this general, it might make sense to extract it to shared styles. (Or not use a utility-style name for the class :) )